tests/operators/test_aws_ecs.py

```diff
@@ -185,6 +185,7 @@ def test_dbt_aes_ecs_overrides_parameter():
 
     assert "containerOverrides" in actual_overrides
     actual_container_overrides = actual_overrides["containerOverrides"][0]
+    assert actual_container_overrides["name"] == "my-dbt-container-name"
```

Comment:
```suggestion
    assert actual_container_overrides["name"] == "dbt"
```
Maybe to maintain consistency with `DEFAULT_CONTAINER_NAME`?

Reply:
Acutally I thought about that, but I believe that this test needs to stay like this, because we want to test out if the actual container value is passed.
therefore we could either add another test, in which we do not set the container_name, and check if it's dbt, and leave this one as it is.